RIPS/rip-7560.md

```diff
@@ -873,6 +512,25 @@ In order to address this developers should track the validity of these transacti
 to rely on the `validUntil` time range parameter to guarantee a transaction that has not been included in the intended time
 will not become valid again unexpectedly for the user who had sent it.
 
+### Denial of Service attacks on the block builder
+
+It is important to consider the ability of the block builder to fill the block with transactions in the allotted time.
+
+In terms of block validity, all validation and execution frames may read and write any state when included in the block.
+This means that transactions are technically able to invalidate each other.
+
+Block builders may face a risk of having multiple eligible
+transactions being invalidated by a single transaction once they include it in a block, causing the Denial of Service.
+Sequencer-like block builders do not have such a risk as long as they process incoming transactions
+in the order they are received.
+
+As part of the mitigation, the AA transactions SHOULD be bound by storage access rules to avoid
+DoS on block builders.
+These rules are defined in [ERC-7562](https://eips.ethereum.org/EIPS/eips/eip-7562).
```

Comment:
remove full paths: just `(./eip-7562)`, regardless if it is eip/erc/ 
The EIP parser should fix it.

Reply:
The RIP repo is not set up with these tools yet, so it will just be a broken link. I think we will be asked to update the document once the parsers and linters are operational.